src/test/java/com/team3/central/services/EventServiceTest.java

```diff
@@ -247,4 +263,98 @@ void getForUser() {
     // then
     assertThat(res).isNotEmpty();
   }
+
+  @Test
+  public void deleteEvent_shouldReturnFalse_whenEventNotFound() {
+    // given
```

Comment:
We need to make naming convention consistent, change this naming style here or for all tests please